recipe/build/CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+cmake_minimum_required(VERSION 3.8)
+project(MediaInfo_CLI VERSION 1.0.0 LANGUAGES CXX)
+
+# Set C++ standard
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+# Include directories
+include_directories(${CMAKE_CURRENT_SOURCE_DIR}/src/Source)
+
+# Source files
+set(CLI_SOURCES
+    ${CMAKE_CURRENT_SOURCE_DIR}/src/Source/CLI/CLI_Main.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/src/Source/CLI/CommandLine_Parser.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/src/Source/CLI/Help.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/src/Source/Common/Core.cpp
+)
+
+# Find required packages
+find_package(ZLIB REQUIRED)
+find_package(Threads REQUIRED)
+find_package(zenlib REQUIRED)
+# find_package(tinyxml2 REQUIRED)
```

Comment:
what is with this package? is it an optional feature? if so, let's add it to `host` and include it